rust/blockstore/src/arrow/blockfile.rs

```diff
@@ -133,12 +133,23 @@ impl ArrowUnorderedBlockfileWriter {
                 Box::new(ArrowBlockfileError::MigrationError(e)) as Box<dyn ChromaError>
             })?;
 
+        let total_keys = self
```

Comment:
We already loop over the blocks above, do we need a second iteration?

Reply:
Is the suggestion to calculate the total before the loop and then iteratively add to the total for each delta?